src/AutoComplete/AutoComplete.component.tsx

```diff
@@ -175,22 +205,26 @@ export const AutoComplete = ({
                         : 'auto-complete'
                 }
                 inputProps={inputProps}
+                type={inputType}
                 value={term}
                 ref={inputRef}
                 size={size}
+                autoComplete={autoComplete ? 'on' : 'off'}
+                autoFocus={autoFocus}
                 prefix={prefix}
                 suffix={suffix}
                 placeholder={placeholder}
                 onFocus={() => setIsFocused(true)}
-                onBlur={() => setIsFocused(false)}
                 onChange={(newValue: string) => {
                     changeSearchTerm(newValue);
                 }}
                 onKeyDown={(event: React.KeyboardEvent) => {
                     switch (event.keyCode) {
                         case EventKeyCodes.TAB:
                         case EventKeyCodes.ENTER:
-                            event.preventDefault();
+                            if (term) {
+                                event.preventDefault();
```

Comment:
Right now if you hit `tab` on an empty input, it picks up the first result. If you don't want that, you could probably do:
```js
if (!term) {
  break;
}
```

Reply:
I kept the same behavior as prod, but if we are fixing the tabbing this makes more sense. 👍